src/components/Header/Header.jsx

```diff
@@ -1,6 +1,12 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Header.css';
+import { Navigation } from '../Navigation/Navigation';
 
-// Create a Header function returning the HTML of header block
-
-// Add a default export statement for Header component to use it in the other files
+export const Header = () => (
+  <>
+    <header className="header">
+      <h1 className="header__title">Site Name</h1>
+      <Navigation />
+    </header>
+  </>
```

Comment:
_comment_
React Fragment is used when you want to have several tags on the same level. It is not needed here.
```suggestion
  <header className="header">
    <h1 className="header__title">Site Name</h1>
    <Navigation />
  </header>
```